resource/inspire2019-prod/profileMetadataDetails.json

```diff
@@ -51,6 +51,15 @@
         "Path": "social.twitter",
         "Value": "",
         "Regex": "@([A-Za-z0-9_]+)$"
+      },
+      {
+        "DisplayTitle": "Website",
+        "FieldName": "Website",
+        "Style": "inputText",
+        "Class": "",
+        "Path": "social.website",
```

Comment:
@gavinbarron  Same with the case here, these changes weren't in the inspire2019-prod version but were in the inspire2019-test version, so, I've moved them over.